src/eigsolve/arnoldi.jl

```diff
@@ -36,11 +36,11 @@ should be targeted. Valid specifications of `which` are
     only be successful if you somehow know that eigenvalues close to zero are also close
     to the periphery of the spectrum.
 
-!!! warning "Degenerate eigenvalues"
+!!! warning "Repeated eigenvalues"
 
     From a theoretical point of view, Krylov methods can at most find a single eigenvector
-    associated with a targetted eigenvalue, even if the latter is degenerate. In the case of
-    a degenerate eigenvalue, the specific eigenvector that is returned is determined by the
+    associated with a targetted eigenvalue, even if the latter is repeated. In the case of
```

Comment:
Maybe we should also clarify the "repeated" terminology for those users who are not familiar
```suggestion
    associated with a targetted eigenvalue, even if the latter is repeated, i.e. the
    eigenvalue has multiple linearly independent eigenvectors or thus an eigenspace
    with dimension (geometric multiplicity) larger than one. In the case of such
```